projects/portal/src/app/pages/faucet/faucet.usecase.service.ts

```diff
@@ -0,0 +1,50 @@
+import { Injectable } from '@angular/core';
+import { Config, ConfigService } from '../../models/config.service';
+import { combineLatest, Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+
+@Injectable({ providedIn: 'root' })
+export class FaucetUseCaseService {
+  config$: Observable<Config | undefined>
```

Comment:
Properties that are not used by others should be kept private.

Reply:
- set config$ private
- Sorry, I noticed that prettier was missing from the default formatter.